packages/components/src/List/Toolbar/ColumnChooserButton/README.md

```diff
@@ -0,0 +1,249 @@
+| submit | function | callback trigger when the form is submit
+| nbLockedLeftItems | number | the number of locked items you want, beginning at the left
+
+To keep the columns from the list sync with the inputs of the user, you will need to merge the data from the column chooser with the columns from the list.
+There is a service to help you to do that, [mergedColumnsChooser](#mergedcolumnschooser).
+
+```javascript
+const columnsMergedWithColumnChooser = {
+		...list,
+		columns: columnChooserService.utils.mergedColumnsChooser(list.columns, columnsChooser),
+};
+<List {...rest} columnChooser={...} list={columnsMergedWithColumnChooser} />;
+```
+If you are using the compound list you just have to import the [ColumnChooserButton](#columnchooserButton) and place it where you need it.
+Do not forget to pass an id in this case.
+
+Either way you will have to update the columns props given to the list with the submitted values.
+
+## Components
+### ColumnChooserButton
+
+The button which will trigger the popover with the form and content.
+If you pass children you will override the default column chooser renderer.
+
+| Props | Type | Info
+|---|---|---|
+| children | React.element | the content of the popover
+| columns | array | these columns comes from the list, they will populate the popover
+| id | string | Use as prefix for all the children components
+| submit | function | callback trigger when the form is submit
+| nbLockedLeftItems | number | the number of locked columns you want, beginning at the left
+| t | function | i18n translate
+
+### ColumnChooser
+
+The base of the popover component.
+This component holds the form. If you pass children you will override the default column chooser renderer.
+The [columnChooserContext](#columnchoosercontext) is initialized here.
+
+The component holds reference :
+* Header : [ColumnChooserHeader](#columnchooserheader)
+* Body: [ColumnChooserBody](#columnchooserbody)
+* Footer :[ColumnChooserFooter](#columnchooserfooter)
+
+| Props | Type | Info
+|---|---|---|
+| children | React.element, [React.element] | the content of the popover
+| columns | array |  these columns comes from the list, they will populate the popover
+| id | string | Use as prefix for all the children components
+| onClose | function | callback trigger when the submit is done.
+| nbLockedLeftItems | number | the number of locked items you want, beginning at the left
+| submit | function | callback trigger when the form is submit, passed to the column manager hook (see hook)
+| t | function | i18n translate
+
+### ColumnChooserHeader
+
+The header of the column chooser.
+Title by default is "Modify columns".
+You can pass children to the header to customize it.
+It consumes the [columnChooserContext](#columnchoosercontext).
+
+| Props | Type | Info
+|---|---|---|
+| className | string | class passed to the tooltip header
+| children | React.element, [React.element] | the header's content of the popover
```

Comment:
is children related to customization at the beginning of the topic?

Reply:
Children allows customization yep.